examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientTls.java

```diff
@@ -61,9 +78,8 @@ public HelloWorldClientTls(String host,
                                SslContext sslContext) throws SSLException {
```

Comment:
Please use a static factory method and have no work involved in the constructor. You never want to throw an exception from a constructor because a constructor must always create an object, even if an exception is thrown.